spread.yaml

```diff
@@ -663,20 +663,9 @@ suites:
                 systemctl status snapd.socket || true
                 journalctl -e
             fi
-    tests/core18/:
-        summary: Subset of core18 specific tests
-        systems: [ubuntu-core-18-*]
-        prepare: |
-            "$TESTSLIB"/prepare-restore.sh --prepare-suite
-        prepare-each: |
-            "$TESTSLIB"/prepare-restore.sh --prepare-suite-each
-        restore-each: |
-            "$TESTSLIB"/prepare-restore.sh --restore-suite-each
-        restore: |
-            "$TESTSLIB"/prepare-restore.sh --restore-suite
-    tests/core20/:
-        summary: Subset of core20 specific tests
-        systems: [ubuntu-core-20-*]
+    tests/core/:
```

Comment:
Unless I'm missing something the suite directory is not renamed. This should probably be included in the patch.

Reply:
Eh, I'm blind :D Sorry, github UI for renames is subtle.